media/css/base/banners/includes/_base.scss

```diff
@@ -2,10 +2,8 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
-$font-path: '/media/protocol/fonts';
-$image-path: '/media/protocol/img';
-
-@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+// NOTE: config must be handled by file @use-ing this includes file
```

Comment:
test with `!default` vars too

Reply:
I don't think I can declare `lib` defaults from the `base` file: https://sass-lang.com/documentation/variables/#configuring-modules